lua/terrortown/entities/roles/sidekick/shared.lua

```diff
@@ -21,7 +22,7 @@ function ROLE:PreInitialize()
 	self.surviveBonus = 1
 	self.scoreKillsMultiplier = 5
 	self.scoreTeamKillsMultiplier = -16
-	self.preventWin = true
+	self.preventWin = GetConVar("ttt2_siki_mode"):GetInt() ~= 3

```

Comment:
This might be problematic because this is only changed when the map is changed and breaks the sidekick with other roles. If for example the clairvoyant has a sidekick, this changes it as well.

Reply:
preventWin should now change as soon as the convar gets changed.